app/assets/stylesheets/layouts/dark.scss

```diff
@@ -1,81 +1,183 @@
-  background-color: #464646;
+
+$bg_color_post_description: #5D5D5D;
+//$font_color_post_description: #F3F3F3;
+$bg_color_navheader: #5F5D57;
+//$bg_color_navbutton: $colors_greybrown;
+
+.post-subheader { background-color: $bg_color_post_description; }
+.post-navheader { background-color: $bg_color_navheader; }
+
+// TODO: this doesn't seem to style #post-menu-box, is that ok?
+
+
+//$bg_color_icon_overlay: #000000; // box displaying current icon
+$bg_color_selectionpopup: #565656; // selection popups like character/alias on posts#editor
+$bg_color_expander: #6F7769; // expander on posts
+
+.selector-popup { background-color: $bg_color_selectionpopup; }
+.selector-popup-button { opacity: 0.7; } /* make the white less contrasty */
+.post-expander { background-color: $bg_color_expander; }
+
+$bg_color_post_icon: #444D3F;
+$bg_color_icon_selector: #444D3F;
+$font_color_icon_selector: #111111;
+
+.post-icon { background-color: $bg_color_post_icon; }
+#current-icon-dropdown { background-color: $bg_color_post_icon; color: $font_color_icon_selector; }
+#reply-icon-selector { background-color: $bg_color_icon_selector; }
+
+$bg_color_post_character: #595959;
+$bg_color_post_screenname: #515151;
+$bg_color_post_author: #4D4C4C;
+//$bg_color_post_character_narrow: $colors_lightbrown;
+//$bg_color_post_author_narrow: $colors_brown; // these are flipped from not-narrow mode to make the quick-switcher blend in // TODO: Should these flip on dark too?
+//$font_color_post_info: #F3F3F3;
+
+.post-character { background-color: $bg_color_post_character; }
+.post-screenname { background-color: $bg_color_post_screenname; }
+.post-author { background-color: $bg_color_post_author; }
+
+// TODO: This theme also seems seems not to style pagination
+
+table { border-color: #555; } // TODO: what does this style
+
+/* - icons */
+//$border_color_icon_selected: #B9BBEB; // selected icon border
+$bg_color_icon: #444D3F;
+$bg_color_icon_credit: #444D3F;
+
+.replace-icon, .icons-box { background-color: $bg_color_icon; }
+.icon-credit { background-color: $bg_color_icon_credit; }
+
+/* - tag items */
+$bg_color_tag_item: #85837C;
+$font_color_tag_item: #DDDDDD;
+
+.tag-item { background-color: $bg_color_tag_item; color: $font_color_tag_item; }
+
+
+/* - link-boxes */
+//$bg_color_linkbox_new: #463C65;
+//$bg_color_linkbox_edit: #463C65;
+//$bg_color_linkbox_fav: #463C65;
+//$bg_color_linkbox_del: #463C65;
+//$bg_color_linkbox_dismiss: #463C65;
+
+.link-box { color: #DDDDDD; }
+
+/* - other buttons */
+$bg_color_viewbutton: #85837C;
+$bg_color_viewbutton_selected: #6A6A6A;
+
+.view-button {
+  background-color: $bg_color_viewbutton;
   color: #DDDDDD;
 }
-.flash {
-  a, a:visited { color: #DDD; }
-  a:hover { color: #BBB; }
-}
-.post-subheader { background-color: #5D5D5D; }
-#post_content, #reply_content {
+.view-button.selected { background-color: $bg_color_viewbutton_selected; }
+
+
+#post_content, #reply_content { // TODO: what does this style?
```

Comment:
doesn't, from a quick search; can be removed (as can it be from `replies.css`)

Reply:
Can this comment be removed?

Reply:
hmm I think that's there for like, needing to pull out those into variables of some kind?

Reply:
It styles the HTML editor for posts and replies – you could simply use the standard input background, or find out if there's a name for the TinyMCE editor's background separately?